cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -127,6 +124,69 @@ func NewFromCluster(client clientset.Interface, certificatesDir string) (*Client
 	return etcdClient, nil
 }
 
+// retrieveEtcdEndpoints returns the list of etcd endpoints.
+func retrieveEtcdEndpoints(client clientset.Interface) ([]string, error) {
```

Comment:
Is there a reason to use `retrieve` instead of `get` like eg for `getAPIEndpoint`?

Reply:
No, I'll change that.